cli/default-reporter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pnpm/default-reporter",
-  "version": "13.0.3",
+  "version": "13.1.3",
```

Comment:
why have you updated this? the version is automatically bumped.

Reply:
Hadn't realized, sorry!